pkgs/by-name/pl/playit-agent/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "playit-agent";
+  version = "0.16.3";
+
+  src = fetchFromGitHub {
+    owner = "playit-cloud";
+    repo = "playit-agent";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-i+v1oyssmeOoMXcyJ8nnS0nTwfKJXXIiIu3KAXBzH1I=";
+  };
+
+  cargoHash = "sha256-Y29Ktb/Vpwdy988pm1PHWmHeHtfsiAGHQLpQ8AZo0L4=";
+
+  meta = {
+    description = "Tunnel client for playit.gg";
+    homepage = "https://playit.gg";
+    changelog = "https://github.com/playit-cloud/playit-agent/releases/tag/v${finalAttrs.version}";
+    license = with lib.licenses; [
+      bsd2
+    ];
+    maintainers = with lib.maintainers; [
+      maxicode
+    ];
```

Write in one line is better.  
```suggestion
    license = lib.licenses.bsd2;
    maintainers = with lib.maintainers; [ maxicode ];
```